src/components/DataTable/DataTable.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useMemo } from 'react';
+import { Table, TableHeader, TableColumn, TableBody, TableRow, TableCell, Pagination } from '@nextui-org/react';
+
+interface Column {
+  label: string;
+  key: string;
+}
+
+interface DataTableProps<T> {
```

Comment:
Props interfaces should go into `/domain/props`

Reply:
done